tasks/src/main/java/org/example/aProblemAboutEquation/AProblemAboutEquation.java

```diff
@@ -0,0 +1,65 @@
+package org.example.aProblemAboutEquation;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.StringTokenizer;
+
+/**
+ * https://codeforces.com/problemset/problem/174/A
+ */
+public class AProblemAboutEquation {
+    float aProblemAboutEquation() {
+        AProblemAboutEquation.FastReader sc = new AProblemAboutEquation.FastReader();
+        int numberOfFriends = sc.nextInt();
+        int millilitersOfTheDrinkInTheBottle = sc.nextInt();
+
+        float sumVolumeOfTheDrinkInAllMugs = 0;
+        List<Integer> volumeOfTheDrinkInTheMugs = new ArrayList<>();
+        for (int i = 0; i < numberOfFriends; i++) {
+            volumeOfTheDrinkInTheMugs.add(sc.nextInt());
+            sumVolumeOfTheDrinkInAllMugs += volumeOfTheDrinkInTheMugs.get(i);
+        }
+
+        List<Float> volumeOfTheDrinkToAdd = new ArrayList<>();
+        float averageVolumeOfTheDrinkInMug = (sumVolumeOfTheDrinkInAllMugs + millilitersOfTheDrinkInTheBottle) / numberOfFriends;
+        for (int i = 0; i < numberOfFriends; i++) {
+            volumeOfTheDrinkToAdd.add(averageVolumeOfTheDrinkInMug - volumeOfTheDrinkInTheMugs.get(i));
+            if (volumeOfTheDrinkToAdd.get(i) < 0) {
+                return -1;
+            }
+        }
+
+        for (int i = 0; i < numberOfFriends; i++) {
+            return volumeOfTheDrinkToAdd.get(i);
```

Comment:
ta petla jest bez sensu bo return idzie od razu na i = 0

Reply:
Rownie dobrze mozna napisac return volumeOfTheDrinkToAdd.get(0)

Reply:
Return dopisałem bez weryfikacji. Pierwotnie zrobiłem BigDecimal, a później podpatrzyłem rozwiązanie i zrobiłem System.out.printf("%f\n"). Chodzi o to, że musi być println i dokładność do 6 miejsc po przecinku aby przejść testy Codeforces.
Oczywiście udało mi się to rozwiązać:  https://codeforces.com/submissions/corvidae

Reply:
rozumiem ze ma byc dokladnosc do 6 miejcs po przecinku, ale ten for jest bezuzyteczny. O to mi chodzi. WIec do usuniecia

Reply:
Dlatego właśnie wczoraj go zmieniłem w czwartym commicie.